replacements/Rendering guide.txt

```diff
@@ -0,0 +1,24 @@
+Please include the following files:
+
+Please make sure you have all the rights to your samples! All original works included should be licensed under the Creative Commons licensing. (Probably under CC BY-SA)
+
+Please put each style in their own folder
+-----------------------
+For unfinished works:
+- Stems in WAV format of each instrument.
+- A "unfinished.txt" document with the following info: Full name of the creator(s), Song name (ID if unknown), Artist name (if any)
+
+
+-----------------------
+For finished works:
+
+- A FLAC file of the finished product. This can be included in the official soundtrack released beside the game. Put thisone in \replacements\soundtrack\
```

Comment:
Typo:
```suggestion
- A FLAC file of the finished product. This can be included in the official soundtrack released beside the game. Put this in \replacements\soundtrack\
```